libs/quasselc/Makefile

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (C) 2016 OpenWrt.org
```

Comment:
It was proposed recently that we should only use this header if there was a legal contract between you and OpenWrt to do the copyright transfer.  Otherwise I would recommend keep the copyright to your self (the author)